includes/reader-revenue/woocommerce/class-woocommerce-connection.php

```diff
@@ -336,7 +336,9 @@ public static function sync_reader_from_order( $order, $verify_created_via = tru
 		if ( ! $contact ) {
 			return;
 		}
-
+		if ( ! method_exists( 'Newspack_Newsletters_Subscription', 'add_contact' ) ) {
```

Comment:
Does this need a `\` prefix since the `Newspack_Newsletters_Subscription` class is in a different namespace than this class?

Reply:
Not really, when passing it as a string argument it doesn't inherit the file's namespace. It'll always require the fully-qualified class name, which in the root doesn't call for `\`.

Reply:
TIL! Thanks for explaining.